### ISSUE TITLE
Adapter draws too much power

### ISSUE THREAD

**Issue by m-kloeckner** [danielkucera/ebus-adapter#7]
Thank you for sending the adapter so quickly! I am able to connect to the ebus of my Wolf heating system with ebusd - no issues with that.

However, the adapter seems to draw too much power from my bus. When only connecting the adapter it works fine, but when attaching the adapter and the heating control unit (Wolf BM), both go off. Unfortunately, I need both to be attached.

Is there a way to provide external power to the ESP12F on the adapter?

**Comment by danielkucera** [comment]
Hi @m-kloeckner , I am sorry to hear that. 

Are you willing to experiment a bit?

Would you mind trying to connect a big capacitor the the ESP power rail? You can experiment with the values, minimum voltage 5V, capacity at least 470uF (bigger should be better but may influence the device startup). E.g. this one: https://www.conrad.de/de/p/yageo-sy006m4700b5s-1330-elektrolyt-kondensator-radial-bedrahtet-5-mm-4700-f-6-3-v-20-x-h-13-mm-x-30-mm-1-st-442651.html

To the connector between 3V3 and GND (pin2 and 3 or 7):
![image](https://user-images.githubusercontent.com/1734256/191006309-84667a9a-ee2d-4189-bf28-e55758659d21.png)

**Comment by m-kloeckner** [comment]
I am willing to experiment! However, I currently don't have such a capacitor at hand and would need to order one. (Or find one in old circuits....)

I also found a second eBus interface on the heating unit. I will try this one later today. Maybe it has separate power supply and allows operation of the adapter. I will keep you updated.

**Comment by m-kloeckner** [comment]
Short update: It works via the second ebus interface.

**Comment by danielkucera** [comment]
Happy to hear that. Thanks for letting me know.

**Comment by m-kloeckner** [comment]
Maybe as an information to others: I also considered adjusting the WiFi transmission power by adding a `WiFi.setOutputPower(x)` statement to the code. But I did not try this out.

It seemed that the power consumption was exceeded once the adapter connected to my AP, thus the idea of limiting WiFi power. Maybe that is worth considering.

**Comment by peterjanto** [comment]
@m-kloeckner Hi! I have the same problem. In my house is installed CGB-2 and BM-2. Which heating unit you have ? I can't find the second ebus interface.
Thx!

**Comment by m-kloeckner** [comment]
@peterjanto I have a CGB-(K)-20 with BM. I found a wiring diagram in the manual (see page 69, Schaltplan) which indicated another eBus interface directly on the control board.

See circle in red:
![image](https://user-images.githubusercontent.com/13656866/202704560-a3150a94-a518-4ef0-ba0d-8c1287a619c9.png)

I tried to find a suitable female connector for this interface without success, so I simply soldered two wires to eBus (-) and eBus (+).

However, it turned out that while this works more reliably, it does not fully eliminate the issue. It works for a while but sporadically crashes. Thus, I gave up on the project. In addition to the power issue, the ebusd config for Wolf is quite incomplete and I did not want to investigate much into the readings.
